content/en/docs/reference/glossary/quantity.md

```diff
@@ -30,3 +30,18 @@ intentionally lowercase), `M` (mega), `G` (giga), `T` (tera), `P` (peta),
 The accepted binary (power-of-2) units are `Ki` (kibi), `Mi` (mebi), `Gi` (gibi),
 `Ti` (tebi), `Pi` (pebi), `Ei` (exbi).
 
+Some terminologies - 
+
+                     `mB` stands for millibyte,
+                     `KB` stands for kilobyte,
+                     `MB` stands for megabyte, 
+                     `GB` stands for gigabyte, 
+                     `TB` stands for terabyte, 
+                     `PB` stands for petabyte, 
+                     `EB` stands for exabyte, 
+                     `KiB` stands for kibibyte, 
+                     `MiB` stands for mebibyte, 
+                     `GiB` stands for gibibyte, 
+                     `TiB` stands for tebibyte, 
+                     `PiB` stands for pebibyte, 
+                     `EiB` stands for exbibyte
```

Comment:
I'd prefer to link to the a web page with the list. This list might make people think that these strings are valid as part of a quantity, but they are not.

Reply:
Hey, can you please elaborate more if possible?